src/lighteval/models/transformers/adapter_model.py

```diff
@@ -105,3 +106,11 @@ def _create_auto_model(self) -> transformers.PreTrainedModel:
         )
 
         return model
+
+    def cleanup(self):
```

Comment:
I could open another PR for this, it was bloating the pipeline code but it's unrelated to the PR